README.md

```diff
@@ -49,6 +49,17 @@ scoop bucket add auth0 https://github.com/auth0/scoop-auth0-cli.git
 scoop install auth0
 ```
 
+Install via [Powershell](https://learn.microsoft.com/en-us/powershell/)
+
+```powershell
+# Download the binary to the current folder. Use the desired version number.
+$version = "1.2.0"
```

Comment:
Is there a way to pull in the latest version instead of hardcoding? 1.2.0 is already out-of-date and I'd prefer not need to update this value whenever we make a release.

Reply:
Maybe we can make this into a powershell script, like the one for linux/mac so user just needs to execute it

Reply:
@deepu105 I like the PowerShell script approach as it matches the Bash script approach. Let's head in that direction.